File: server/src/main/java/de/hsrm/mi/swtpro03/server/network/messages/editor/EditorInitMessage.java
package de.hsrm.mi.swtpro03.server.network.messages.editor;

import de.hsrm.mi.swtpro03.server.game.dtos.MapDTO;

public class EditorInitMessage {

	public static final String TYPE = "EditorInitMessage";
	private MapDTO mapDTO;

	public EditorInitMessage() {
	}

	public EditorInitMessage(MapDTO mapDTO) {
		this.mapDTO = mapDTO;
	}

	public MapDTO getMapDTO() {
		return mapDTO;
	}

	public void setMapDTO(MapDTO mapDTO) {
		this.mapDTO = mapDTO;
	}
}

File: client/Client/Network/Messages/Editor/EditorInitMessage.cs
using LoadRunnerClient.DTOs;

namespace LoadRunnerClient.Messages
{
    /// Author Florian Ortmann
    public class EditorInitMessage
    {
        public const string TYPE = "EditorInitMessage";

        private MapDTO _mapDTO;

        public EditorInitMessage(MapDTO mapDTO)
        {
            this._mapDTO = mapDTO;
        }

        public MapDTO mapDTO
        {
            get => _mapDTO;
            set => _mapDTO = value;
        }
    }
}